### PR TITLE
Prevent uwu output from pinging globally allowed roles.

### PR DESCRIPTION
**Pull request by scragly**
Description

This PR introduces sanitisation of the uwu text before uwufication in order to prevent moderator roles being pinged unnecessarily.

I've just changed the `str` typehint to the `clean_content` converter to do this as simply as possible.

## Reasoning

While we do have a global limit to allowed roles being pinged in the bot, all mod+ roles were still capable of being pinged through this method.

## Screenshots

**Before** (role resolves)
image

**After** (role is now plaintext)
image

## Did you:

- [X] Join the [**Python Discord Community**](https://discord.gg/python)?
- [X] If dependencies have been added or updated, run `pipenv lock`?
- [X] **Lint your code** (`pipenv run lint`)?
- [x] Set the PR to **allow edits from contributors**?

### CODE REVIEW

File: bot/exts/evergreen/fun.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ async def roll(self, ctx: Context, num_rolls: int = 1) -> None:
         await ctx.send(output)
 
     @commands.command(name="uwu", aliases=("uwuwize", "uwuify",))
-    async def uwu_command(self, ctx: Context, *, text: str) -> None:
+    async def uwu_command(self, ctx: Context, *, text: clean_content(fix_channel_mentions=True)) -> None:
```
Comment (SebastiaanZ) [inline bot/exts/evergreen/fun.py:74]: If the command argument is a message ID or message jumplink, the command will internally fetch the message text on this line in the command:

```py
        text, embed = await Fun._get_text_and_embed(ctx, text)
```

This bypasses the `clean_content` converter (which is only applied to the ID/jumplink, not the message text that is fetched from it later).

**Review by SebastiaanZ — CHANGES_REQUESTED** [review]

File: bot/exts/evergreen/fun.py (hunk 1 above)
Comment (scragly) [reply bot/exts/evergreen/fun.py:74]: Good point, thanks. I'll get that fixed up.

Comment (scragly) [reply bot/exts/evergreen/fun.py:74]: OK, I should have resolved this now, along with also applying the cleaning to the mixed case command so mentions don't scramble the raw mention format, instead always using the resolved names.

**Review by SebastiaanZ — APPROVED** [review]

**Review by Akarys42 — APPROVED** [review]
Works like a charm

**Pull request merged**